spec/services/people/finder_spec.rb

```diff
@@ -16,5 +16,13 @@
         expect(people_finder.call.pluck(:id)).to eq [person.id]
       end
     end
+
+    context 'when matching nomis_offender_no filter' do
```

Comment:
I'm not a fan of this pattern (relying on data set in the factory in the test) as it's not very self-describing (Yes I know that's what the other example did).